src/main/java/nl/tudelft/ti2206/group9/level/CrashMap.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.ti2206.group9.level;
+
+import nl.tudelft.ti2206.group9.level.entity.AbstractEntity;
+import nl.tudelft.ti2206.group9.level.entity.AbstractObstacle;
+import nl.tudelft.ti2206.group9.level.entity.AbstractPickup;
+import nl.tudelft.ti2206.group9.level.entity.Player;
+import nl.tudelft.ti2206.group9.level.entity.PowerupInvulnerable;
+import nl.tudelft.ti2206.group9.util.CollisionMap;
+import nl.tudelft.ti2206.group9.util.GameObserver;
+
+import static nl.tudelft.ti2206.group9.ShaftEscape.OBSERVABLE;
+
+/**
+ * Add all the collision handlers to a CollisionMap.
+ * @author Mathias
+ */
+public class CrashMap {
```

Comment:
Maybe you should consider renaming this class to `CollisionHandler` or something like that.
(I know I suggested the current name, but it is not really clear)